src/content/advanced/vsphere-kubevip/index.md

```diff
@@ -0,0 +1,92 @@
+---
+linkTitle: vSphere Kube-vip
+title: vSphere networking with Kube-vip
+description: Here we describe how to work with Kube-vip in Giant Swarm CAPV clusters.
+weight: 25
+menu:
+  main:
+    parent: advanced
+last_review_date: 2023-10-09
+user_questions:
+  - How to divide my subnet for Kube-vip with CAPV?
+  - What IP should I use for Kube-vip with CAPV?
+  - How to set a range of IPs for load balancer in Kube-vip with CAPV?
+  - Where can I see the IPs in use by Kube-vip?
+aliases:
+  - /guides/kube-vip-for-capv/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+While most infrastructure providers offer a load balancer to use for the Kubernetes API and services of type Load Balancer (AWS, Azure, VMware Cloud Director...), vSphere is a bit different in that it is a hypervisor as opposed to a cloud provider. Advanced networking use cases such as routing, firewalling, load balancing and the likes are handled by VMware NSX. In order to offer a highly-available Kubernetes API even to customers without NSX, CAPV comes with `kube-vip`, a layer-2 load balancer.
+
+As a result of working on layer 2, `Kube-vip` uses IP addresses in the same subnet as the nodes themselves and these IP addresses must be selected carefully to avoid collisions. This is particularly plausible if multiple users collaborate on the platform and select the same IP for different clusters.
+
+## IPAM controller
+
+in order to mitigate this risk, Giant Swarm CAPV clusters run `cluster-api-ipam-provider-in-cluster` to track IP addresses usage from within the management cluster (MC). For that reason, **customers shouldn't run more than one management cluster per subnet**.
```

Comment:
```suggestion
In order to mitigate this risk, Giant Swarm CAPV clusters run `cluster-api-ipam-provider-in-cluster` to track IP address usage from within the management cluster (MC). For that reason, **customers shouldn't run more than one management cluster per subnet**.
```

Reply:
But this is something customer run or we do for them?

Reply:
This is us, we added it to the cluster chart